.github/workflows/book.yml

```diff
@@ -27,8 +29,9 @@ jobs:
           cargo install mdbook --version "^0.4" --locked
           echo "${HOME}/.cargo/bin" >> $GITHUB_PATH
 
+      # Removed --locked for now since it is broken due to old proc_macro feature.
       - name: Install linkchecker
-        run: cargo install mdbook-linkcheck --version "^0.7" --locked
+        run: cargo install mdbook-linkcheck --version "^0.7"
```

Comment:
Is it `mdbook-linkcheck` that is broken? Is there an issue that we could link to?

Reply:
The issue is with `proc_macro`. It's the issue we've seen many times before when a feature gets stabilized but the current nightly hasn't caught up. I don't have an issue, but I also don't think it's worth it at this point.

Worst case scenario, the CI will break and we can readjust.